content/cloud-docs/run/cli.mdx

```diff
@@ -49,7 +49,9 @@ To enable the CLI-driven workflow, you must:
 
 1. Run `terraform login` to authenticate with Terraform Cloud. Alternatively, you can [manually configure credentials in the CLI config file](/cli/config/config-file#credentials).
 
-1. Add the `cloud` block to your Terraform configuration. The example below shows how you can map your CLI workspaces to Terraform Cloud workspaces that have a specific tag.
+1. Add the `cloud` block to your Terraform configuration. You can define its arguments direcly in your configuration file or supply them through for [non-interactive workflows](#non-interactive-workflows). Refer to [Using Terraform Cloud](/cli/cloud) for configuration details.
```

Comment:
```suggestion
1. Add the [`cloud`](https://www.terraform.io/cli/cloud/settings) block to your Terraform configuration. You can define its arguments directly in your configuration file or supply them through environment variables, which can be useful for [non-interactive workflows](#non-interactive-workflows). Refer to [Using Terraform Cloud](/cli/cloud) for configuration details.
```

Reply:
not sure if we need full path for the link though

Reply:
We already have this link in the "refer to" text, so I'm going to remove the one you added in the first sentence. Otherwise, I approve!